openapi/openapi_v2_resource_test.go

```diff
@@ -484,6 +484,12 @@ func TestBuildResourceNameFromPath(t *testing.T) {
 			expectedResourceName: "cdns",
 			expectedError:        nil,
 		},
+		{
+			path:                 "/cdns-test",
+			preferredName:        "",
+			expectedResourceName: "cdns-test",
```

Comment:
For consistency reasons and to abide by Terraform snake_case naming conventions, can the expectedResourceName be updated to cdns_test?

Reply:
But the dash will be replaced by an underscore by the plugin ?
Currently the tests are passing so I am a little bit confused ...

Reply:
yeah that'd be a good option, perhaps replacing the dashes with underscores and updating the tests to reflect that.

Reply:
Done